courses/machine_learning/deepdive2/supplemental/solutions/question_answering_rnn.ipynb

```diff
@@ -0,0 +1,684 @@
+{
```

Comment:
<ul><li>pep8: Two blank lines between functions.</li><li>"F1_score" --&gt; "f1_score"</li><li>"F1" --&gt; "f1"</li><li>In general, I'd prefer to use already implemented functions to compute the metrics, as in sklearn.metric</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1325/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Because we're doing a F1 Score on string data, this one unfortunately does not exist as is in sklearn!</p>